packages/arb-validator/checkpointing/indexedCheckpointer.go

```diff
@@ -40,6 +41,8 @@ var errNoMatchingCheckpoint = errors.New("cannot restore because no matching che
 type IndexedCheckpointer struct {
 	*sync.Mutex
 	db                    machine.CheckpointStorage
+	bs                    machine.BlockStore
+	ns                    machine.NodeStore
```

Comment:
I'll change the variable name, but I'd rather leave the struct name as is, since there's nothing about it that's fundamentally linked to a node being confirmed, though that's certainly the only use case at the moment.